### PR TITLE
[draft] dhcp stress test with copp policy

### PR DESCRIPTION
**Pull request by yyynini**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
The purpose of this test is to conduct a DHCP stress test with CoPP policy, ensuring that the DHCP relay service can withstand the maximum load applied to the CPU

#### How did you do it?
In the test, we will generate packets at a rate of 10,000 packets per second for 120 seconds to ensure the DHCP relay service can handle the maximum load without failure.

#### How did you verify/test it?
In the test log file, we will record the number of packets sent and received via the discover, offer, request, and ack stages, noting that all of them are sent by broadcast. Meanwhile, we can manually check the device to monitor the status of the DHCP container, resource utilization, and verify that the DHCP processes are running.

### CODE REVIEW

**Comment by yyynini** [comment]
image

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -325,6 +325,12 @@ dhcp_relay/test_dhcp_relay.py::test_dhcp_relay_unicast_mac:
     conditions:
       - "'dualtor' in topo_name and release in ['201811', '201911']"
 
+dhcp_relay/test_dhcp_reply_stress_with_copp.py:
```
Comment (yyynini) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:328]: typo here

File: tests/dhcp_relay/test_dhcp_reply_stress_with_copp.py [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+
+    if testing_mode == DUAL_TOR_MODE:
+        skip_release(duthost, ["201811", "201911"])
+
+    if len(dut_dhcp_relay_data) > 1:
+        pytest.skip("skip the unicast mac testcase in the multi-Vlan setting")
+
+    for dhcp_relay in dut_dhcp_relay_data:
+        # Run the DHCP relay test on the PTF host
+        breakpoint()
```
Comment (yyynini) [inline tests/dhcp_relay/test_dhcp_reply_stress_with_copp.py:523]: remove this

File: tests/dhcp_relay/test_dhcp_relay_stress_with_copp.py [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+    is_backend_topology
+):
+    """
+        Pytest fixture to handle setup and cleanup for the COPP tests.
+    """
+    upStreamDuthost = None
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    test_params = _gather_test_params(tbinfo, duthost, request, duts_minigraph_facts)
+
+    if not is_backend_topology:
```
Comment (yaqiangz) [inline tests/dhcp_relay/test_dhcp_relay_stress_with_copp.py:338]: No need, dhcp_relay wouldn't be enabled in backend tor

File: tests/dhcp_relay/test_dhcp_relay_stress_with_copp.py [hunk 4]
```diff
@@ -0,0 +1,585 @@
...
+    enum_rand_one_per_hwsku_frontend_hostname,
+    creds,
+    ptfhost,
+    tbinfo,
+    duts_minigraph_facts,
+    request,
+    is_backend_topology
+):
+    """
+        Pytest fixture to handle setup and cleanup for the COPP tests.
```
Comment (yaqiangz) [inline tests/dhcp_relay/test_dhcp_relay_stress_with_copp.py:332]: Does this test support in dualtor?

**Review by yaqiangz — CHANGES_REQUESTED** [review]

**Comment by yaqiangz** [comment]
`draft` in title could be removed?

**Pull request closed without merging**